apps/discord-bot/src/commands/gamecounts.command.ts

```diff
@@ -0,0 +1,72 @@
+import { INFO_COLOR } from '#constants';
+import { ApiService, Page, PaginateService } from '#services';
+import { Command, CommandContext, EmbedBuilder } from '@statsify/discord';
+import { prettify } from '@statsify/util';
+
+@Command({
+  description: (t) => t('commands.gameCounts'),
+})
+export class GameCountsCommand {
+  public constructor(
+    private readonly apiService: ApiService,
+    private readonly paginateService: PaginateService
+  ) {}
+
+  public async run(context: CommandContext) {
+    const counts = await this.apiService.getGameCounts();
+    const resource = await this.apiService.getResource('games');
+    const gameInfo = resource?.games;
```

Comment:
Don't use `gameInfo` for game names. It is fine for sub game names tho. I hate how it makes `BedWars` into `Bed Wars`